From e6775fc7d910e426a5b1f200f23d2e640e828477 Mon Sep 17 00:00:00 2001
From: wolbernd <wolbernd@users.noreply.github.com>
Date: Tue, 24 Aug 2021 12:03:43 +0200
Subject: [PATCH] Fixed issue #17535: Error when logging in with LDAP

Function str_pos() does not exist in PHP -> changed to strpos()
---
 application/core/plugins/AuthLDAP/AuthLDAP.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/application/core/plugins/AuthLDAP/AuthLDAP.php b/application/core/plugins/AuthLDAP/AuthLDAP.php
index ea0c4d49cb..32164db399 100644
--- a/application/core/plugins/AuthLDAP/AuthLDAP.php
+++ b/application/core/plugins/AuthLDAP/AuthLDAP.php
@@ -330,7 +330,7 @@ private function createConnection()
         }
 
         // Try to connect
-        if (str_pos($ldapserver, 'ldaps://') === false && str_pos($ldapserver, 'ldap://') === false) {
+        if (strpos($ldapserver, 'ldaps://') === false && strpos($ldapserver, 'ldap://') === false) {
             $ldapserver = 'ldap://' . $ldapserver;
         }
         $ldapconn = ldap_connect($ldapserver . ':' . (int) $ldapport);
